ui/main/src/app/modules/monitoringexp/monitoringexp.component.ts

```diff
@@ -0,0 +1,315 @@
+        ['information', 4]
+    ]);
+    private processMonitoring: any[];
+    selectedCardId: string;
+
+    constructor(
+        private processesService: ProcessesService,
+        private configService: ConfigService,
+        private dateTimeFormatter: DateTimeFormatterService,
+        private cardService: CardService,
+        private translationService: TranslationService,
+        private entitiesService: EntitiesService,
+        private modalService: NgbModal,
+        private selectedCardService: SelectedCardService
+    ) {
+        this.processMonitoring = this.configService.getConfigValue('processMonitoring');
+
+        processesService.getAllProcesses().forEach((process) => {
+            if (process.uiVisibility?.monitoring_exp) {
+                const itemName = process.name ? process.name : process.id;
+                this.processNames.set(process.id, itemName);
+                for (let key of process.states.keys()) {
+                    this.processStateDescription.set(process.id + '.' + key, process.states.get(key).description);
+                    this.processStateName.set(process.id + '.' + key, process.states.get(key).name);
+                    this.stateColors.set(process.id + '.' + key, process.states.get(key).color);
+                }
+
+                this.listOfProcessesForRequest.push(process.id);
+                this.listOfProcessesForFilter.push({
+                    value: process.id,
+                    label: itemName,
+                    i18nPrefix: `${process.id}.${process.version}`
+                });
+            }
+        });
+    }
+
+    ngOnInit() {
+        this.size = this.configService.getConfigValue('monitoringexp.filters.page.size', 10);
+        this.tags = this.configService.getConfigValue('monitoringexp.filters.tags.list');
+
+        this.results = [];
+
+        this.interval = setInterval(() => {
+            if (this.currentPage === 0) {
+                this.sendFilterQuery(0);
+            } else {
+                this.sendFilterQuery(this.currentPage - 1);
+            }
+        }, 5000);
+
+        this.selectedCardService.getSelectCardIdChanges().subscribe(selectedCardId => this.selectedCardId = selectedCardId)
+    }
+
+    ngAfterViewInit() {
+        this.isProcessGroupFilterVisible = this.filtersTemplate.isProcessGroupFilterVisible();
+        this.sendFilterQuery(0);
+    }
+
+    resetForm() {
+        this.monitoringExpForm.reset();
+        this.columnFilters = [];
+        this.firstQueryHasBeenDone = false;
+    }
+
+     sendFilterQuery(page_number): void {
+        this.technicalError = false;
+        this.loadingInProgress = true;
+
+        const {value} = this.monitoringExpForm;
+        this.filtersTemplate.transformFiltersListToMap(value);
+
+        const filter = this.getFilter(page_number, this.size, this.filtersTemplate.filters);
+
+        this.cardService
+            .fetchFilteredCards(filter)
+            .pipe(takeUntil(this.unsubscribe$))
```

Comment:
You do not need to unsubscribe as the observable is close automatically in this case

Reply:
Done. Removed.